<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>979</width>
    <height>702</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>979</width>
    <height>702</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Google Street View</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="locationEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="viewButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
background-color: qlineargradient(spread:pad, x1:0.989, y1:0.33, x2:1, y2:0, stop:0 rgba(216, 216, 216, 255), stop:1 rgba(154, 154, 154, 255));
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px; 
}
QPushButton:pressed 
{
color: rgb(70, 70, 70);
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
 }
QPushButton:hover 
{
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
}
</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>578</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QLabel" name="fovLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>The horizontal field of view of the image:</string>
      </property>
     </widget>
     <widget class="QLabel" name="headingLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>The compass heading of the camera:</string>
      </property>
     </widget>
     <widget class="QLabel" name="pitchLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Up or down angle of the camera:</string>
      </property>
     </widget>
     <widget class="QLabel" name="headingValueLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="fovValueLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pitchValueLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="history">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background: rgb(230, 230, 230) ;</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="historyTab">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QPushButton" name="removeButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>312</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: qlineargradient(spread:pad, x1:0.989, y1:0.33, x2:1, y2:0, stop:0 rgba(216, 216, 216, 255), stop:1 rgba(154, 154, 154, 255));
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px; 
}
QPushButton:pressed 
{
color: rgb(70, 70, 70);
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
 }
QPushButton:hover 
{
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
}
</string>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeAllButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>312</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string>QPushButton 
{
background-color: qlineargradient(spread:pad, x1:0.989, y1:0.33, x2:1, y2:0, stop:0 rgba(216, 216, 216, 255), stop:1 rgba(154, 154, 154, 255));
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px; 
}
QPushButton:pressed 
{
color: rgb(70, 70, 70);
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
 }
QPushButton:hover 
{
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
}
</string>
      </property>
      <property name="text">
       <string>Remove All</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="historyTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>295</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>27</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>27</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="defaultButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>312</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: qlineargradient(spread:pad, x1:0.989, y1:0.33, x2:1, y2:0, stop:0 rgba(216, 216, 216, 255), stop:1 rgba(154, 154, 154, 255));
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px; 
}
QPushButton:pressed 
{
color: rgb(70, 70, 70);
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
 }
QPushButton:hover 
{
background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 30), stop:1 rgba(154, 154, 154, 255));
}
</string>
      </property>
      <property name="text">
       <string>Load Default</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="browser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>631</width>
      <height>639</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="browserTab">
     <attribute name="title">
      <string>Browser</string>
     </attribute>
     <widget class="QWebView" name="webView">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>9</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="control">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="controlTab">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QDial" name="headingDial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="fovSlider">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>41</width>
        <height>111</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSlider" name="pitchSlider">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>41</width>
        <height>111</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>-90</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter an address here:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="copyrightLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>Logo</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/logo.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
